content/resources/faq.md

```diff
@@ -5,18 +5,23 @@ toc = "true"
 
 ## A Note on all questions and answers
 
-This FAQ is a good faith attempt by the GitBOM community to gather the questions that have been asked and answers to them.
+This FAQ is a good-faith attempt by the GitBOM community to answer common questions that we have been asked. It is intended to evolve over time and as the community grows.
 
-This FAQ is intended to evolve and improve as more viewpoints from more community members are brought to bear.
+To propose a new question, please (open an [issue](https://github.com/git-bom/site/issues)). 
 
-Additional questions can and should be proposed (open an [issue](https://github.com/git-bom/site/issues)). 
-
-Question answers pairs should improve (pushing a [PR](https://github.com/git-bom/site/pulls)).   
+To propose a question-answer pair, please (open a [pull request](https://github.com/git-bom/site/pulls)).   
 
 ## Gitref's are based on sha1, isn't sha1 broken?
 
 [shattered.io](https://shattered.io/) has broken sha1 in PDF files. Additional work has been done to create inexpensive [chosen prefix] collisions.
 
+## Git has been considering moving to sha256 for some time. Why doesn't GitBOM simply adopt that?
+
+Great question. We might.
+
+## Why isn't information about the compiler/linker included in the GitBOM?
+
+Our view is that build environment information which does not affect the build output should not be represented in the GitBOM, as doing so would invalidate the characteristics of **Uniqueness** and **Artifact Identity**.
```

Comment:
We might want to make these links, as we are using them as terms of art here without explanation.

Reply:
They are defined in the very next section of the FAQ -- and I agree, both references should be links. It's probably worth pulling out the 10 attributes, listed in the WP, into something on the site with shortlinks.

Reply:
Markdown titles automatically get links and TOC entries...